docs/development/docguide.rst

```diff
@@ -114,9 +114,9 @@ command. Provided that Sphinx is installed, the above command will temporarily
 install sphinx-astropy_ and all its dependencies automatically.
 
 .. _astropy-helpers: https://github.com/astropy/astropy-helpers
-.. _NumPy: http://www.numpy.org/
-.. _numpydoc: https://pypi.python.org/pypi/numpydoc/0.3.1
-.. _Matplotlib: http://matplotlib.org/
+.. _NumPy: https://www.numpy.org/
+.. _numpydoc: https://pypi.org/project/numpydoc/0.3.1
```

Comment:
beyond this PR, but this link needs to be updated to be version agnostic (we are well beyond 0.3.1 now)